nautobot/utilities/api.py

```diff
@@ -30,6 +30,20 @@ def get_serializer_for_model(model, prefix=""):
         )
 
 
+def get_filterset_class_for_model(model):
```

Comment:
I'm not clear on why this was added, seeing as we already have `nautobot.utilities.utils.get_filterset_for_model()`? Is it just to raise the `FilterSetNotFound` exception? If so, I'd rather see the calling code just raise this error explicitly if it calls the existing `get_filterset_for_model()` and gets `None` in return.